README.md

```diff
@@ -122,23 +122,24 @@ The cert-manager-csi driver aims to have complete feature parity with all
-| `csi.cert-manager.io/reuse-private-key`  | Re-use the same private when when renewing certificates.                                              | `false`            | `true`                           |
+| Attribute                                | Description                                                                                           | Default                              | Example                          |
+|------------------------------------------|-------------------------------------------------------------------------------------------------------|--------------------------------------|----------------------------------|
+| `csi.cert-manager.io/issuer-name`        | The Issuer name to sign the certificate request.                                                      |                                      | `ca-issuer`                      |
+| `csi.cert-manager.io/issuer-kind`        | The Issuer kind to sign the certificate request.                                                      | `Issuer`                             | `ClusterIssuer`                  |
+| `csi.cert-manager.io/issuer-group`       | The group name the Issuer belongs to.                                                                 | `cert-manager.io`                    | `out.of.tree.foo`                |
+| `csi.cert-manager.io/common-name`        | Certificate common name.                                                                              |                                      | `my-cert.foo`                    |
+| `csi.cert-manager.io/dns-names`          | DNS names the certificate will be requested for. At least a DNS Name, IP or URI name must be present. |                                      | `a.b.foo.com,c.d.foo.com`        |
+| `csi.cert-manager.io/ip-sans`            | IP addresses the certificate will be requested for.                                                   |                                      | `192.0.0.1,192.0.0.2`            |
+| `csi.cert-manager.io/uri-sans`           | URI names the certificate will be requested for.                                                      |                                      | `spiffe://foo.bar.cluster.local` |
+| `csi.cert-manager.io/duration`           | Requested duration the signed certificate will be valid for.                                          | `720h`                               | `1880h`                          |
+| `csi.cert-manager.io/is-ca`              | Mark the certificate as a certificate authority.                                                      | `false`                              | `true`                           |
+| `csi.cert-manager.io/key-usgaes`         | Set the key usages on the certificate request.                                                        | `digital signature,key encipherment` | `server auth,client auth`        |
```

Comment:
typo `usgaes`

Reply:
Should the example value include the defaults too? E.g. `digital signature,key encipherment,server auth,client auth` 
And does the parser allow spaces after each comma?

Reply:
What I mean is, is it ever valid to *not* include `digital signature,key encipherment`....I always see those in the certificates I use.

Reply:
Though should be the defaults now.
It doesn't expect a space after the commas no. 
> What I mean is, is it ever valid to not include digital signature,key encipherment....I always see those in the certificates I use.
Not sure about the last- I am doing the default of cert-manager and matching the behavior with choosing usages.

Reply:
I have updated to remove the spacing around commas